packages/liveblocks-react/.eslintrc.js

```diff
@@ -8,7 +8,23 @@ module.exports = {
     // -------------------------------
     // Custom syntax we want to forbid
     // -------------------------------
-    "no-restricted-syntax": ["error", ...commonRestrictedSyntax],
+    "no-restricted-syntax": [
+      "error",
+      ...commonRestrictedSyntax,
+      // TODO: Enable the following error once React 19 is released and our polyfill is ready
+      // {
+      //   selector:
+      //     "ImportDeclaration[source.value='react'] ImportSpecifier[imported.name='use']",
+      //   message:
+      //     "use is only available on React >=19. Import it from '/src/lib/use-polyfill' instead.",
+      // },
+      {
+        selector:
+          "ImportDeclaration[source.value='react'] ImportSpecifier[imported.name='useSyncExternalStore']",
+        message:
+          "useSyncExternalStore is only available on React >=18. Import it from 'use-sync-external-store/shim/index.js' instead.",
+      },
```

Comment:
Should these be the same as in the other `.eslintrc` file?

Reply:
For `useTransition` and `useId` we have polyfills only in `react-ui` not here because we don't use them, I guess ultimately it's the same question as above with `use`, should we error out even if there's no alternative way (yet)? Maybe that's wiser, since it would prevent all cases, not just ones that have been dealt with before and led to a polyfill.

Reply:
Updated in https://github.com/liveblocks/liveblocks/pull/1774/commits/e99a0e3e2e5738c8bb570d491282c436582233d7.